content/post/yet-another-reason-to-privatize-the-police.md

```diff
@@ -9,43 +9,60 @@ title = "The time has come to privatize the police"
 +++
 
 
-In a matter of hours the entire nation was made aware of the gruesome and undeniably wrongful death of George Floyd, a 46 year old man who was [suffocated](https://twitter.com/QasimRashid/status/1265389213714915333) to death for [using a forged $20 bill](https://www.thecut.com/2020/05/man-pinned-down-by-minneapolis-police-officer-dies.html) at Cup Foods, a small grocery store in Minneapolis, Minnesota. 
+In a matter of hours the entire nation was made aware of the gruesome and undeniably wrongful death of George Floyd, a 46 year old man who was [suffocated](https://twitter.com/QasimRashid/status/1265389213714915333) to death for [using a forged $20 bill](https://www.thecut.com/2020/05/man-pinned-down-by-minneapolis-police-officer-dies.html)[^1] and [refusing to return](https://en.as.com/en/2020/05/31/other_sports/1590954856_526126.html) his stolen goods to
+Cup Foods, a small grocery store in Minneapolis, Minnesota. 
 
-This tragedy refocused national attention to police brutality, a problem which sadly permeates much of our nation's police forces. This pattern of behavior has been captured by The Free Thought Project, which has publicized [more than 600 pages](https://thefreethoughtproject.com/category/cop-watch/) of incidents in which police officers are caught grossly abusing their power. 
+This tragedy has unequivocally refocused national attention on the problem of police brutality, a well-known feature of the current police system.[^2] Social media quickly became laced with dozens, if not hundreds of videos capturing horrific abuses of power in the wake of protests catalyzed by Floyd's death. I am not talking so much about the aggressive and unconventional law enforcement measures taken against belligerent rioters and aggressive thieves, but those taken against peaceful protestors and non-violent citizens for crimes like [voicing discontent](https://twitter.com/sweeeetdee_/status/1267319103167107072?s=20) with local law enforcement or [filming police from your private property](https://www.usatoday.com/story/news/nation/2020/06/02/george-floyd-protest-minneapolis-cops-shoot-paint-people-porch/3123781001/). These are only two of the many instances of badge abuse that have occurred in just the past couple of weeks. Far more disturbing, however, is The Free Thought Project's collection of [more than 600 *pages*](https://thefreethoughtproject.com/category/cop-watch/) of incidents in which police officers are caught grossly abusing their power. 
```

Comment:
Not sure about the wording above.  Made an attempt.  You may want to rework.

Reply:
I liked your wording better